packages/mobile/src/config.ts

```diff
@@ -55,6 +55,7 @@ export const SMS_RETRIEVER_APP_SIGNATURE = Config.SMS_RETRIEVER_APP_SIGNATURE
 // ODIS minimum dollar balance for pepper quota retrieval
 // TODO change this to new ODIS minimum dollar balance once deployed
 export const ODIS_MINIMUM_DOLLAR_BALANCE = 0.1
+export const VERIFICATION_STATE_EXPIRY_SECONDS = 30
```

Comment:
does this relate to the expiry the attestation request?  i'm guess no considering this is so short

Reply:
No, when user open goes to initial verification screen - we fetch current verification state. Then user decides on what to do and go to verification itself. If user were "deciding" for more than `VERIFICATION_STATE_EXPIRY_SECONDS` - we would refetch verification status before going to actual verification. I will add it as a comment in the code